packages/aws-cdk-lib/aws-codedeploy/test/server/deployment-group.test.ts

```diff
@@ -212,6 +213,70 @@ describe('CodeDeploy Server Deployment Group', () => {
     });
   });
 
+  test('can be created with multiple ALB Target Groups as the load balancers', () => {
```

Comment:
```suggestion
  test('can be created with multiple Target Groups as the load balancers', () => {
```
You can specify also CLBs and NLBs.